### ISSUE TITLE
The classes provided by the pre-trained model of vipseg_stdc(v1,v2) do not match the ground truth

### ISSUE THREAD

**Issue by Iriswwhy** [lxtGH/Tube-Link#11]
When I use the pretrained models using stdc backbone(tb_link_vipseg_stdcv1.pth,tb_link_vipseg_stdcv2.pth) to get inference results of my own data, the labels on the visualization result are wrong. For example, "rail" is the label of the mask of "car". Is there something wrong in the stdc pretrained models?

**Comment by lxtGH** [comment]
Hi! Thanks for using our method. According to your message, I cannot judge the errors are from. Maybe you can try the stronger model like Swin-Large.
